install-dev.sh

```diff
@@ -1,5 +1,17 @@
 #!/bin/sh
+
+# Check if pip is installed, and install it if necessary
+echo "---- Updating pip ----"
+curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py
+python get-pip.py
+echo "---- Pip updated ----"
+rm get-pip.py
+if ! pip --version > /dev/null 2>&1; then
+    echo "Installing pip..."
+    curl -sSL https://bootstrap.pypa.io/get-pip.py | python
+fi
 echo "---- Installing Python Poetry ----"
```

Comment:
Would have put that line down ahah

Reply:
Are you going to merge it?

Reply:
Yes ! We have to resolve the comments though ☺️